saas/views/download.py

```diff
@@ -421,3 +424,273 @@ def queryrow_to_columns(self, record):
             self.encode(transaction.dest_unit),
             self.encode_descr(transaction)
         ]
+
+class BalancesMetricsDownloadView(CSVDownloadView, BalancesMetricsMixin):
```

Comment:
Mixin should be before View (i.e. View is always the last class in the inheritance)

`class BalancesMetricsDownloadView(BalancesMetricsMixin, CSVDownloadView):`